Cartfile

```diff
@@ -1,2 +1,2 @@
 github "pusher/pusher-platform-swift" "v1"
-github "pusher/push-notifications-swift" ~> 3.0.1
+github "pusher/push-notifications-swift" ~> 3.0.4
```

Comment:
Don't we want to remove the Cartfile altogether if `pusher-platform-swift` and `pusher-notifications-swift` are now coming from SPM?

Reply:
This file was used for 2 different reasons:
 - To define public dependencies of our SDK for everyone integrating using Carthage. This is still required.
 - To provide us with `PusherPlatform` and `PushNotifications` frameworks when doing the development of `PusherChatkit`. This is no longer required because Xcode fetches dependencies using SPM.

Reply:
> To define public dependencies of our SDK for everyone integrating using Carthage. This is still required.
Aha, cool, wasn't aware of this one. Thanks for educating me.